index.html

```diff
@@ -180,6 +187,31 @@ <h3>
           [=computed MIME type=] of an [=image resource=].
         </p>
       </section>
+      <section>
+        <h3>
+          <dfn>label</dfn> member
+        </h3>
+        <p>
+          The {{ImageResource/label}} of an {{ImageResource}} is a string that
+          provides an accessible name for the associated image.
+        </p>
+        <p>
+          The {{ImageResource/label}} is strongly recommended, but optional as
```

Comment:
Recommended already implies optional. 

```suggestion
          The {{ImageResource/label}} is strongly recommended as
```

Reply:
+1 to @marcoscaceres' proposed change.